dolphin/analysis/output.py

```diff
@@ -237,6 +237,13 @@ def plot_model_overview(self, lens_name, model_id=None,
         :return: `matplotlib.pyplot.figure` instance with the plots
         :rtype: `matplotlib.pyplot.figure`
         """
+        if print_Results:
```

Comment:
```
if print_results:
            if kwargs_result is None:
                print(self.load_output(lens_name, model_id)["kwargs_result"])
            else:
                print(kwargs_result)
```

Reply:
That what I initially wrote too, but I change it as to test that else statement I would need to  would  reduce test  provide a sensible kwargs_result  dict, which I think will bloat `test_output.py`. This is consistent  with` get_model_plot` at line 179